tesseract/tesseract_geometry/include/tesseract_geometry/mesh_parser.h

```diff
@@ -44,16 +47,20 @@ TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
 #include <assimp/scene.h>
 #include <assimp/Importer.hpp>
 #include <assimp/postprocess.h>
+#include <assimp/pbrmaterial.h>
```

Comment:
I think we could do something similar to what is done for tesseract_kinematics [here](https://github.com/ros-industrial-consortium/tesseract/blob/e9d6289a864997eb21bea8c0c93343e1404fb0c9/tesseract/tesseract_kinematics/CMakeLists.txt#L84) where we check the version and if less than 5.0.0 set a compiler definition and use a default pbr settings.

Reply:
I added conditional building to disable PBR with older versions of ASSIMP.